packages/server/src/data/db/db-operations/post-ops.js

```diff
@@ -61,10 +61,15 @@ const getPostsByFilter = async ({
   return Post.aggregate(aggregationPipeline);
 };
 
+const getPostByID = async (id) => {
+  return await Post.findById(id);
```

Comment:
Should prob project the same fields as above except with comments

Reply:
u right